<?xml version="1.0"?>

<launch>
  <arg name="num_samples" default="1000" />
  <arg name="chain_start" default="pelvis" />
  <arg name="chain_end" default="leftThumbRollLink" />
  <arg name="timeout" default="0.005" />

  <param name="robot_description" textfile="$(find val_description)/model/urdf/valkyrie_sim.urdf"/>


  <node name="ik_solver" pkg="val_manipulation" type="val_ik_solver_node" output="screen">
    <param name="num_samples" value="$(arg num_samples)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="/robot_description"/>
  </node>


</launch>
